dev-libs/cdk/files/cdk-5.0.20240619-xlib.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/cdk-config.in b/cdk-config.in
```

Comment:
Can you put a description at the top of the patch too (just copy the commit message tbh)?